lametro/bills.py

```diff
@@ -76,7 +76,7 @@ def session(self, action_date) :
         localize = pytz.timezone(self.TIMEZONE).localize
         fmt = '%Y-%m-%d'
 
-        for session in Lametro.legislative_sessions:
+        for session in Lametro.legislative_sessions.fget():
```

Comment:
What is `fget()` doing for you?

Reply:
When I switched to the property decorator, what's returned from this method is a property object, which seems like it can't be operated on as is. I thought it'd work similarly to a property decorator on a model where it just gives you the value, but this `fget()` is the only way I've found so far to get the value when it's just a regular class.

```
(Pdb) Lametro.legislative_sessions
<property object at 0xffff99656de0>
(Pdb) Lametro.legislative_sessions.fget()
[{'identifier': '2014', 'start_date': '2014-07-01', 'end_date': '2015-06-30'},  ..., {'identifier': '2024', 'start_date': '2024-07-01', 'end_date': '2025-06-30'}]

(Pdb) type(Lametro.legislative_sessions)
<class 'property'>
(Pdb) type(Lametro.legislative_sessions.fget())
<class 'list'>
```